vuepress/docs/v7.0/docs/README.md

```diff
@@ -5,28 +5,56 @@ sidebarDepth: 0
-
-## Why Kubernetes?
-
-Instead of you doing the work to ensure your apps and services stay up and running, Kubernetes does the work for you. For each app or service, you can set resource limits, the number of pods you want backing each service, upgrade strategy, and auto-scaling. Kubernetes actively manages your cluster to match your defined resource utilization and ensures your site doesn't go down.
-
-::: tip How Google Runs Containers in Production
-Each week, Google deploys over 2 billion containers in production to run all of its services like Gmail, YouTube, and Search. Kubernetes is Google's open source solution to automate the management of containers at scale.
-:::
-
-Over 50% of Fortune 500 companies use Kubernetes with product offerings from Amazon Elastic Kubernetes Service (EKS), Google Cloud Platform (GKE), IBM (Red Hat) OpenShift, Microsoft Azure (AKS), and VMWare Pivotal (PKS).
+It is a distributed and composable environment in which components can be built, scaled and iterated independently, helping teams build agile applications that can adapt to change.
+
+![Entando Platform Development Process](./getting-started/img/concepts.png) 
+
+On the Entando Platform, development of an application is organized into four stages: Create, Curate, Compose, and Consume. In the simplest terms, you create functional building blocks, curate them into a catalog, compose the blocks onto an app or website, which is then consumed by a user.
+
+## Create Components 
+The first step in building a modern application on Entando is to design and create each independent component or functional block. Developers can use the [Entando Component Generator](../create/component-gen-overview.md) and [Blueprint](../create/blueprint-features.md) to automate the building and packaging of project files, with the flexibility to customize while using software they are accustomed to. The modular design of apps built on Entando means new business capabilities can be integrated into your application without reworking existing systems. 
+
+| Documentation | Tutorials        |
+| :------------ | :--------- |
+| [Entando Component Generator](../create/component-gen-overview.md) | [Micro Frontends](../../tutorials/create/mfe/) |
+| [Entando Blueprint Feature](../create/blueprint-features.md) | [Microservices](../../tutorials/create/ms/generate-microservices-and-micro-frontends.md) |
+| [Component Generation Technologies](../create/component-gen-tech.md) | [Build and Publish Project Bundles](../../tutorials/create/pb/publish-project-bundle.md) |
+
+## Curate and Share Components 
+On the Entando Platform, Curate and Share means development teams can bundle and share components within a central catalog. Multiple teams can share and collaborate on a single component or whole packaged business capabilities (PBCs) across organizations and applications. They can test and assess components for readiness, manage and publish versions, and share metadata for continuous upgrades. 
```

Comment:
Curate and Share -> just Curate -- we never use Share outside of the left nav, and Curate implies/includes that in the descriptions

"Multiple teams can share and collaborate on a single component..." -> "Multiple teams can collaborate on and share a single component..." --reverse order because chronologically you collaborate before sharing

remove "whole" -- unnecessary; a packaged business capability is by definition the whole kit and caboodle. you would never refer to a component as part of a business capability - instead you'd use the component name.

avoid "they"... users? teams? "they" is awkward when referring to a group here; "they" reads like people performing actions but refers to groups